semantics/language/translation/decl/global.k

```diff
@@ -437,19 +444,19 @@ module C-DECL-GLOBAL
      // differ in the case of declarations without initializers. A naked
      // declaration without an initializer constitutes a tentative definition,
      // but an extern declaration without an initializer does not.
-     syntax KItem ::= "declareExternalVariable'" "(" K "," DeclState "," Linkage "," Storage ")"
-          [strict(1)]
+     syntax KItem ::= "declareExternalVariable'" "(" Bool "," K "," DeclState "," Linkage "," Storage ")"
```

Comment:
The initializer coming back empty after calculating it doesn't necessarily mean it was an empty initializer to begin with because it's the "frozen" initializer that's returned. If there was nothing to freeze (it could all be run at translation-time) it'll just be .K.